### ISSUE TITLE
Project Report: Column Sort not working in all columns.

### ISSUE THREAD

**Issue by slylandro** [partkeepr/PartKeepr#349]
It appears column sorting isn't working in the Part Name, Part Description, Distributor, columns by clicking on the column title bar.
##

**Comment by felicitus** [comment]
Depends on #452

**Comment by Addy771** [comment]
I'd like to add to this that the sorting doesn't work for storage location either. I'm running 0.82 and I have storage locations such as "A03" or "C12" and these don't get sorted at all

**Comment by felicitus** [comment]
@Addy771 are you referring to storage location sorting in the project reports?

**Comment by Addy771** [comment]
Oh, didn't notice this referred to project reports. I am experiencing this in the Parts List

**Comment by felicitus** [comment]
@Addy771 can you open a separate issue including instructions on how to reproduce?

**Comment by Addy771** [comment]
Alright
